astro-airflow-iris/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/settings.py

```diff
@@ -22,8 +22,9 @@
 # CONF_SOURCE = "conf"
 
 # Class that manages how configuration is loaded.
-# from kedro.config import OmegaConfigLoader
-# CONFIG_LOADER_CLASS = OmegaConfigLoader
+from kedro.config import OmegaConfigLoader  # noqa: E402
```

Comment:
https://github.com/kedro-org/kedro-starters/pull/145/files I see that using the rule name works, do you want to apply the same change here?